hyperspy/tests/mva/test_decomposition.py

```diff
@@ -229,3 +253,38 @@ def test_decomposition_supported_return_false(self):
                 algorithm=algorithm,
                 return_info=False,
                 output_dimension=1) is None
+
+
+@pytest.mark.skipif("sys.platform == 'darwin'")
+class TestPlotExplainedVarianceRatio:
+
+    def setup_method(self, method):
+        MY_PATH = os.path.dirname(__file__)
```

Comment:
yes, it did this initially, but the plotting test failed because the data (and therefore the explained variance ratios) are not exactly the same!! And it needs a big enough dataset to have enough components.
Would it be better if I open the data from a numpy file? Or is there a way to always get the same explained variance ratios from a generated data?